cpp/ql/lib/semmle/code/cpp/ir/implementation/internal/TIRVariable.qll

```diff
@@ -2,21 +2,21 @@ private import TIRVariableInternal
 private import Imports::TempVariableTag
 
 newtype TIRVariable =
-  TIRUserVariable(Language::Variable var, Language::LanguageType type, Language::Function func) {
+  TIRUserVariable(Language::Variable var, Language::LanguageType type, Language::Declaration func) {
```

Comment:
```suggestion
  TIRUserVariable(Language::Variable var, Language::LanguageType type, Language::Declaration decl) {
```
Also in a few cases below. I'll stop commenting on this now. I just think this is more consistent. I don't have any problems off-loading this in a separate PR with is just some consistency fixes.